docs/user/dql/metadata.rst

```diff
@@ -36,21 +36,18 @@ Example 1: Show All Indices Information
 
 SQL query::
 
-	POST /_plugins/_sql
```

Comment:
Were these tests not run before? Why does the output missing the `employees_nested` table_name?

Reply:
They never run before! `employees_nested` table is in IT, not in DT.